tools/coverage/calc/delta.go

```diff
@@ -112,16 +112,18 @@ func (changes *GroupChanges) processChangedFiles(
 		pathFromProfile := githubUtil.FilePathProfileToGithub(inc.base.Name())
 		fmt.Printf("checking if this file is in github change list: %s", pathFromProfile)
 		if (*githubFilePaths)[pathFromProfile] == true {
-			fmt.Printf("\tYes!\n")
+			fmt.Printf("\tYes!")
```

Comment:
nit: for these cases, use `fmt.Println`.

Reply:
I intentionally don't want to print a newline until later